test/ClassMethodsHydratorTest.php

```diff
@@ -46,18 +46,18 @@ public function testCanExtractFromMethodsWithOptionalParameters(): void
      */
     public function testCanHydratedPromiscuousInstances(): void
     {
-        /** @var ClassMethodsCamelCase $classMethodsCamelCase */
```

Comment:
removed for `UnnecessaryVarAnnotation`
https://github.com/laminas/laminas-hydrator/runs/3177583829?check_suite_focus=true